### PR TITLE
Provide correct fuse create file status

### PR DESCRIPTION
**Pull request by LuQQiu**
### What changes are proposed in this pull request?

Provide correct FUSE create file status when creating the file.

### Why are the changes needed?

For S3 and many object storage, files are not visible until the writing is finished.
But for the POSIX standard, they expect the file to be immediately visible after creation.
e.g. create() - getattr() - write() - write() - release()
the getattr() in the middle would fail for S3 and many object storage because file cannot be listed before it finished written.

### Does this PR introduce any user facing changes?
Yeah, create then getattr() can pass with S3 and many object storage

### CODE REVIEW

File: integration/fuse/src/main/java/alluxio/fuse/file/FileStatus.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.fuse.file;
+
+/**
+ * The file status of ongoing fuse stream.
+ */
+public class FileStatus {
+  private long mFileLength;
```
Comment (yyongycy) [inline integration/fuse/src/main/java/alluxio/fuse/file/FileStatus.java:18]: Only one field?

**Comment by yyongycy** [comment]
this is to move everything to a feature branch?

File: integration/fuse/src/main/java/alluxio/fuse/file/FileStatus.java (hunk 1 above)
Comment (LuQQiu) [reply integration/fuse/src/main/java/alluxio/fuse/file/FileStatus.java:18]: Yeah, for read now only one field is needed. This is added to simplify the FuseFileStream interface

**Comment by LuQQiu** [comment]
> this is to move everything to a feature branch?

This previously contains changes from other PRs, now it's cleaned

**Comment by LuQQiu** [comment]
@yyongycy @ssz1997 PTAL, thanks!

**Review by ssz1997 — APPROVED** [review]
LGTM

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**